packages/python/javdb_integrations/rclone_manager.py

```
@@ -448,6 +452,30 @@ def export_dedup_history() -> int:
+    Returns the total number of rows updated across both tables.
+    """
+    from packages.python.javdb_platform.db import get_db, OPERATIONS_DB_PATH
+
+    updated = 0
+    with get_db(OPERATIONS_DB_PATH) as conn:
+        for table, col in [('RcloneInventory', 'FolderPath'),
+                           ('DedupRecords', 'ExistingGdrivePath')]:
+            cur = conn.execute(
+                f"UPDATE {table} SET {col} = SUBSTR({col}, INSTR({col}, ':') + 1) "
+                f"WHERE INSTR({col}, ':') > 0 "
+                f"AND (INSTR({col}, '/') = 0 OR INSTR({col}, ':') < INSTR({col}, '/'))"
```

Ruff (0.15.7)

[error] Possible SQL injection vector through string-based query construction

(S608)